pkg/app/sensor/app.go

```diff
@@ -477,15 +478,32 @@ func getCurrentPaths(root string) (map[string]interface{}, error) {
 				return filepath.SkipDir
 			}
 
-			if info.Mode().IsRegular() &&
-				!strings.HasPrefix(pth, "/proc/") &&
-				!strings.HasPrefix(pth, "/sys/") &&
-				!strings.HasPrefix(pth, "/dev/") {
-				pth, err := filepath.Abs(pth)
-				if err == nil {
-					pathMap[pth] = nil
+			if !info.Mode().IsRegular() {
```

Comment:
This will prevent the check below from executing as directories is a Mode Bit - which ends up exiting here and prevents line 497-503 from begin useful. I ran tests moving that statement above this check and it worked as expected.

Reply:
Great catch! Pushing a potential fix!